style.css

```diff
@@ -0,0 +1,117 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;500;600;700;800&family=Poppins&family=Roboto:wght@700&display=swap');
+
+html {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #c7c7c7;
+  padding: 0;
+  margin: 0;
+  overflow-y: auto;
+}
+
+.icon {
+  float: right;
+  padding: 20px;
+}
+
+.wrapper {
+  max-height: 90vh;
+  width: 375px;
+  display: flex;
+  flex-direction: column;
+  background-color: inherit;
+  margin: 5vh auto 5vh auto;
+}
+
+.header-wrapper {
+  background-image: url(./Images/illustration-header-mobile.png);
+  background-color: #1c1a19;
+  background-repeat: no-repeat;
+  color: white;
+  background-position-x: 225px;
+  background-position-y: 420px;
+}
+
+.nav-links {
+  display: none;
+}
+
+.wrapper > * {
+  flex: 1 1 auto;
+}
+
+.header-section {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 20px 24px;
+  background: #3c3a39;
+}
+
+.nav-span {
+  padding-top: 10px;
+  cursor: pointer;
+}
+
+.first-section {
```

Comment:
- [ ] Notice that there is a background image in this first section. Look at the picture below 👇 
![image](https://user-images.githubusercontent.com/90222110/176560132-825e0f17-3041-4a7c-948e-c962c821a88d.png)
It is not appearing in your design and we have to stick to the design as much as possible and also it is a the important skill as a developer to pay attention on all details. Please add that image on your design too.